documentation/frontend/reference/i18n.md

```diff
@@ -78,7 +75,7 @@ processed and loaded into Nuxt via the Nuxt i18n module.
 ## Test locales
 
 Three locales are maintained in the code base (as opposed to downloaded from
-Glotpress) for use in testing, alongside the default English locale. These
+GlotPress) for use in testing, alongside the default English locale. These
```

Comment:
tip: You can add GlotPress to the Vale `TermCasing.yaml` so these errors are found automatically in the future.